lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -238,22 +240,24 @@ class LighthouseReportViewer {
 
   /**
    * Updates the page's HTML with contents of the JSON file passed in.
-   * @param {File} file
-   * @return {Promise<void>}
+   * @param {string} str
    * @throws file was not valid JSON generated by Lighthouse or an unknown file
    *     type was used.
    * @private
    */
-  _onFileLoad(file) {
-    return this._readFile(file).then(str => {
-      let json;
-      try {
-        json = JSON.parse(str);
-      } catch (e) {
-        throw new Error('Could not parse JSON file.');
-      }
+  _onFileLoad(str) {
+    let json;
+    try {
+      json = JSON.parse(str);
+    } catch (e) {
+      logger.error('Could not parse JSON file.');
```

Comment:
```suggestion
      logger.error('Could not parse JSON file.');
      return;
```

if we're matching previous behavior